mmcv/cnn/bricks/upsample.py

```diff
@@ -47,6 +49,8 @@ def forward(self, x):
         return x
 
 
+# Avoid BC-breaking of importing build_upsample_layer.
+# It's backwards compatible with the old `build_upsample_layer`.
```

Comment:
why we deprecate the build_upsample_layer

Reply:
The build_upsample_layer (build_upsample_layer_from_cfg) is set as the build_func of UPSAMPLE_LAYERS.